fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -30,8 +30,8 @@ def self.handle(args, options)
 
       if FastlaneCore::FastlaneFolder.swift?
         disable_runner_upgrades = options.disable_runner_upgrades || false
-
-        Fastlane::SwiftLaneManager.cruise_lane(lane, lane_parameters, dot_env, disable_runner_upgrades: disable_runner_upgrades)
+        swift_server_port = options.swift_server_port || 2000
```

Comment:
I think we should also remove this `|| 2000` and put it into as the default in the `cruise_lane` method